pandas/compat/numpy/__init__.py

```diff
@@ -15,6 +15,7 @@
 _np_version_under1p13 = _nlv < LooseVersion('1.13')
 _np_version_under1p14 = _nlv < LooseVersion('1.14')
 _np_version_under1p15 = _nlv < LooseVersion('1.15')
+_np_version_equal1p15 = _nlv == LooseVersion('1.15')
```

Comment:
Is the pattern to change this to 
```
_np_version_under1p16 = _nlv < LooseVersion('1.16')
_np_version_equal1p16 = _nlv == LooseVersion('1.16')
```
when numpy 1.16 comes out? Or is this a one off to meet a current need?